CesiumAsync/src/ResponseCacheControl.cpp

```diff
@@ -14,17 +14,25 @@ ResponseCacheControl::ResponseCacheControl(
     bool accessControlPublic,
     bool accessControlPrivate,
     bool proxyRevalidate,
-    int maxAge,
-    int sharedMaxAge)
+    bool maxAgeExists,
+    int maxAgeValue,
```

Comment:
Consider using `std::optional<int>` instead of the bool/int pairing.

Reply:
Done